src/content/tutorials/fleet-management/job-management/_index.md

```diff
@@ -0,0 +1,485 @@
+  values: |
+    waitForPodsReady:
+      enable: true
+      timeout: 10m
+```
+
+Now update the app resource to point to the configmap:
+
+```yaml
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  name: <CLUSTER_NAME>-kueue
+  namespace: org-fer
+spec:
+  ...
+  userConfig:
+    configMap:
+      name: <CLUSTER_NAME>-kueue-userconfig
+      namespace: org-<ORGANIZATION>
+```
+
+Now define two new queues (Cluster and Local) to run the tests:
+
+```yaml
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ClusterQueue
+metadata:
+  name: gang-cluster-queue
+spec:
+  namespaceSelector: {}
+  resourceGroups:
+  - coveredResources: ["cpu", "memory", "nvidia.com/gpu"]
+    flavors:
+    - name: default-flavor
+      resources:
+      - name: "cpu"
+        nominalQuota: 10
+      - name: "memory"
+        nominalQuota: 20Gi
+    - name: gpu-flavor
+      resources:
+      - name: "nvidia.com/gpu"
+        nominalQuota: 2
+---
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: LocalQueue
+metadata:
+  namespace: default
+  name: gang-queue
+spec:
+  clusterQueue: "gang-cluster-queue"
+```
+
+Now, you create a `JobSet` to define a driver controller and some workers:
+
+```yaml
+apiVersion: jobset.x-k8s.io/v1alpha2
+kind: JobSet
+metadata:
+  name: gang-jobset
+  namespace: default
+spec:
+  replicatedJobs:
+  - name: driver
+    replicas: 1
+    template:
+      metadata:
+        labels:
+          kueue.x-k8s.io/queue-name: gang-queue
+      spec:
+        parallelism: 1
+        completions: 1
+        template:
+          spec:
+            containers:
+            - name: driver
+              image: busybox:latest
+              command: ["sh", "-c", "echo 'Driver job running'; sleep 60"]
+              resources:
+                requests:
+                  cpu: 1
+                  memory: 1Gi
+            restartPolicy: Never
+  - name: workers
+    replicas: 3
+    template:
+      metadata:
+        labels:
+          kueue.x-k8s.io/queue-name: gang-queue
+      spec:
+        parallelism: 2
+        completions: 2
+        template:
+          spec:
+            containers:
+            - name: worker
+              image: busybox:latest
+              command: ["sh", "-c", "echo 'Worker job running'; sleep 45"]
+              resources:
+                requests:
+                  cpu: 2
+                  memory: 2Gi
+                  nvidia.com/gpu: 1
+            restartPolicy: Never
+            tolerations:
+            - key: nvidia.com/gpu
+              value: "true"
+              effect: NoSchedule
+```
+
+Once you submit the `JobSet` it will create two `ReplicatedJobs` which at the same time will create three worker replicas with two jobs for each. Since those jobs request way more memory, cpu and GPU resources the job will not be scheduled and the whole group will be requeue after time out. You can try to relax the requests and see how Kueue controller schedule the jobs all together when the capacity is available.
```

Comment:
```suggestion
Once you submit the `JobSet`, it will create two `ReplicatedJobs`, which, in turn, will make three worker replicas with two jobs assigned to each. Since those jobs require way more memory, CPU, and GPU resources, the job will not be scheduled, and the whole group will be requeued after a timeout. You can relax the requests and see how the Kueue controller schedules the jobs altogether when capacity is available.
```